### DISCUSSION TITLE
在中文文档中为什么没有对如何设置明亮和黑暗进行介绍？

### DISCUSSION THREAD

**Discussion by jiapy97** [zeit-ui/react#624] [Q&A] [answered]
![image](https://user-images.githubusercontent.com/57215916/132085057-05a02f1f-fec6-4e4b-8d10-6e1a54bef9ae.png)

**Comment by ofekashery** [comment, marked as answer]
Hi @qq1120637483, please see [themes](https://react.geist-ui.dev/zh-cn/guide/themes).

**Reply by jiapy97** [reply]
okay thank you
